### DISCUSSION TITLE
New trigger - Player linked/connected a discord account

### DISCUSSION THREAD

**Discussion by YTG2315** [gameserverapp/Platform#656] [Suggestions]
I would love the ability to award my community for linking there discords to the platform. For example if they were to join the server and run !linkdcgsa if would give them a code that they can use to link to the site. Or just in general give them rewards for linking there accounts to the website!

I would love to see some rcon commands for the shop also. Don't get me wrong GSA is amazing and i love all the work max and the team has done. I would love to have to the ability to add points using rcon - non consumable wise As much i love gsa i absolutely hate PayPal's seller protection especially digital protection its basically non existent.

Thanks for all the work max! its been amazing coming for a unmodded ASE experience!

**Comment by mvaessen** [comment]
Should be possible for sure, but also quite easy to exploit. All the player would need to do is connect & disconnect with various accounts to collect a large number of rewards. So I recommend keeping that in mind

**Reply by YTG2315** [reply]
**Understood thanks for the reply max!**

**Reply by ScaredDonut** [reply]
A workaround could be to have registrationRewardGranted and xxServiceRewardGranted true/false in the database to keep track if the person has already received their reward?

**Reply by babsgosgens** [reply]
I was also going to suggest to make the connecting an event/achievement. It will help onboarding and the process seems rather stressful. Most of our players are ps5 and for them, the connecting has not been without hickups. I feel I should reward them for the effort they put in, all the tme being scared they would get kicked.

The solution by @ScaredDonut to prevent seems doable, right?

**Reply by Glorio53** [reply]
**_**i would love to see some rcon commands to add directly to someone's balance!**_**

**Comment by mvaessen** [comment]
Added to #517

**Comment by mvaessen** [comment]
Hi, this is now available on the dashboard!
